substrate/bin/minimal/node/src/chain_spec.rs

```diff
@@ -0,0 +1,66 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use runtime::{BalancesConfig, RuntimeGenesisConfig, SudoConfig, SystemConfig, WASM_BINARY};
+use sc_service::{ChainType, Properties};
+use sp_keyring::AccountKeyring;
+
+/// Specialized `ChainSpec`. This is a specialization of the general Substrate ChainSpec type.
```

Comment:
Second sentence is just a verbose repetition of the first. I suggest either remove it, or replace it with something more rich with information.

```suggestion
/// Specialized `ChainSpec`.
```